.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,7 +1,6 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
 contact_links:
   - name: Usage question
-    url: https://stackoverflow.com/questions/tagged/python-xarray
-    about: "Post a question on Stack Overflow using the #python-xarray
-    tag. These are regularly reviewed by xarray's maintainers, and questions which
-    include a reproducible example will receive a response."
\ No newline at end of file
+    url: https://github.com/pydata/xarray/discussions
+    about: Ask questions and discuss with other community members here.
```

Comment:
Should we specify the reproducible examples here, or is there a separate template for discussions?